xen/arch/arm/domain.c

```diff
@@ -715,6 +716,9 @@ int arch_domain_create(struct domain *d,
     if ( (rc = tee_domain_init(d, config->arch.tee_type)) != 0 )
         goto fail;
 
+    if ( (rc = ffa_domain_init(d, config->arch.flags)) != 0 )
```

Comment:
I think the plan was to use a new value in tee_type here instead of introducing flags